### ISSUE TITLE
Dark theme / night -mode

### ISSUE THREAD

**Issue by Feijk** [zhukov/webogram#1100]
I implemented a dark theme / night -mode -setting for Telegram's web-version. I've been using it myself for a while now and everything seems to be working. I'll make a PR in case there is any interest for this feature.

The branch: https://github.com/CodeDistillery/webogram/tree/dark-theme

**Comment by migueloop** [comment]
Would you attach a screenshot?

**Comment by Feijk** [comment]
![webogram_dark_04](https://cloud.githubusercontent.com/assets/4600141/15651641/84dd5c5e-2689-11e6-95a3-8a1dae4d0f39.png)
![webogram_dark_03](https://cloud.githubusercontent.com/assets/4600141/15651640/84db46c6-2689-11e6-8ef9-1f5b8d6d1828.png)
![webogram_dark_02](https://cloud.githubusercontent.com/assets/4600141/15651638/84da3ace-2689-11e6-85ea-e6199e9d76ce.png)
![webogram_dark_01](https://cloud.githubusercontent.com/assets/4600141/15651639/84da4866-2689-11e6-9923-0bbab4ecd75d.png)

Here are some screenshots when running as a Chrome App. Most info greyed out, but the main idea should be visible.

**Comment by kimlop** [comment]
there is a plan to merge this code in the master branch?

**Comment by stek29** [comment]
That's what Stylish was made for.

**Comment by digitalheir** [comment]
Just in case anyone might find it useful. It seems webogram is no longer under active development and we won't get native styling. To give people options.

I made a Tampermonkey / Greasemonkey plugin that manipulates the CSS for Telegram Web.

https://gist.github.com/digitalheir/bcbc4e3fe582854399026bef75b658a4

![image](https://user-images.githubusercontent.com/178797/32040839-5fa4dea6-ba32-11e7-9c5c-75bf22991800.png)

**Comment by stek29** [comment]
@digitalheir 
> webogram is no longer under active development 

Zhukov is server-side dev :(

**Comment by Termuellinator** [comment]
why is this not implemented? stylish/greasmonkey don't really work for something like Rambox :/

**Comment by cecton** [comment]
> there is a plan to merge this code in the master branch?

> why is this not implemented? stylish/greasmonkey don't really work for something like Rambox :/

Yeah actually... what it hasn't been merge already?? There is a toggle button and everything. Is it just because the author didn't open a PR but an issue with a link to his branch?

If that's the case I can open a PR.

**Comment by deandreamatias** [comment]
@Feijk you can open a PR? If you can't, maybe @cecton can open.
Thnaks

**Comment by cecton** [comment]
@deandreamatias I forked, made a PR and fixed the merge conflicts. I have not tested though. (I don't mind testing it but not today.)

**Comment by roccomuso** [comment]
+1

**Comment by alex-mayorga** [comment]
¡Hola @zhukov!

Could you outline the happy path forward for this issue, please?

My retinas would thank you =)

¡Gracias!
Alex

**Comment by jovancosta** [comment]
> ¡Hola @zhukov!
> 
> Could you outline the happy path forward for this issue, please?
> 
> My retinas would thank you =)
> 
> ¡Gracias!
> Alex

same here this is a must go feature.
please, just release this.

**Comment by jameskitt616** [comment]
Any time soon this will be merged?

**Comment by varecki** [comment]
Hi @zhukov!
Is it any chance to implement dark mode?

**Comment by migueloop** [comment]
@zhukov any change you democratize this project a little bit more so people that are really interested can merge updates from the community?

**Comment by wellington1993** [comment]
@zhukov what blocks this merge? Who can make merge requests?  Thanks

**Comment by Daverogk** [comment]
On a general note,we need to have a dark theme for the application/webpage ASAP
